### PR TITLE
Fix modals to show on dashboard only

### PR DESCRIPTION
**Pull request by rafa-garcia**
CONTEXT: This changeset fixes a bug where the initial modals were showing on pages beyond the dashboard.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: core/templates/core/base.html [hunk 1]
```diff
@@ -127,14 +127,13 @@
         },
       })
 
-      const loggedInUser = {% if request.user.is_authenticated %}true{%else %}false{% endif %};
-      if (loggedInUser) {
+      {% if tour %}
```
Comment (webbyfox) [inline core/templates/core/base.html:130]: Am I right, `tour` is restricted to only Logged in User?

Comment (rafa-garcia) [reply core/templates/core/base.html:130]: Indirectly, yes. Tour in the context is available only trough wagtail as a logged-in page

Comment (webbyfox) [reply core/templates/core/base.html:130]: we should have better logic to present the `segment` as I am lost to see how `tour` and `segment` is connected.

**Review by webbyfox — APPROVED** [review]
I am approving based on there is some refactoring required as `tour` and `segment` logic need to differentiate.

**Pull request merged**